src/js/components/SkipLinks/SkipLinks.js

```diff
@@ -1,71 +1,64 @@
-import React, { createRef, cloneElement, Component } from 'react';
+import React, { useRef, useState, cloneElement } from 'react';
 
 import { Box } from '../Box';
 import { Heading } from '../Heading';
 import { Layer } from '../Layer';
 
-class SkipLinks extends Component {
-  static defaultProps = {
-    messages: {
-      skipTo: 'Skip To',
-    },
-  };
+const SkipLinks = ({ children, id, messages }) => {
+  const [showLayer, setShowLayer] = useState(false);
+
+  const layerRef = useRef(null);
 
-  state = {
-    showLayer: false,
+  const onFocus = () => {
+    setShowLayer({ showLayer: true });
```

Comment:
```suggestion
    setShowLayer( true );
```
Using hooks allows this further cleanup.